README.md

```diff
@@ -4,15 +4,47 @@ This repo specifies the web backend and frontend for the [Orcasound app](http://
 
 ## Contributing
 
-Please check out the CONTRIBUTING doc for how we work on things and learning resources!
+Please check out the [CONTRIBUTING](CONTRIBUTING.md) doc for tips on making a successful contribution, as well as learning resources!
 
-## Requirements
+## Getting started
 
-Orcasite uses [PostGIS](http://postgis.net/) for location data inside of Postgres. To install on MacOS, run
+##### Quick: `docker-compose pull && docker-compose up`
```

Comment:
One small hiccup was that I had to go to my `/etc/hosts` folder for the right hostname that we'd set up a long time ago for the right docker machine. I suggest adding a step that says something like "get the IP of your default docker machine via `docker-machine ls`, then go to that IP with port `4000`, i.e. `<docker machine ip>:4000`.

Reply:
I think listing Docker as a prerequisite covers this (make sure it works, e.g. you can connect to the daemon and `docker ps`), and also I don't think we should officially support VM/`docker-machine` based setups because they're no longer the recommended way (nowadays I assume most people will be using regular Docker in linux or Docker for Mac/Windows). I see it as a power user thing, so if you're using `docker-machine` you probably know the gotchas already.

BTW as a note for the setup you and I have, you can actually map the localhost port to the VM in virtualbox so that you can use http://localhost:port for the URL. See https://stackoverflow.com/a/10076989 and https://stackoverflow.com/a/32175164